### PR TITLE
Skip warm reboot tests for standalone topologies

### PR DESCRIPTION
**Pull request by ccroy-arista**
Summary:
Skip warm reboot tests for standalone topologies.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311
- [x] 202405

#### What is the motivation for this PR?
Warm reboot is not currently supported in standalone topos. Skipping the corresponding tests for now.

#### How did you do it?
Applied skip conditions to the applicable test in the tests mark conditions yaml file in sonic-mgmt.

#### How did you verify/test it?
Ran the associated tests and confirmed that the tests are now skipped.

#### Any platform specific information?
Verified on Arista-7060X6-64PE-256x200G.

### CODE REVIEW

**Review by yutongzhang-microsoft — APPROVED** [review]

**Comment by Janetxxx** [comment]
LGTM

**Review by Janetxxx — APPROVED** [review]
LGTM

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -53,11 +53,11 @@ arp/test_unknown_mac.py:
...
   skip:
-    reason: "Warm reboot is broken on dualtor topology. Device fails to recover by sanity check. Skipping for now."
+    reason: "Warm reboot is broken on dualtor topology. Device fails to recover by sanity check. Skipping for now. Not yet supported in TH5"
+    conditions_logical_operator: or
     conditions:
-      - "'dualtor' in topo_name"
-      - https://github.com/sonic-net/sonic-buildimage/issues/16502
-
+      - "https://github.com/sonic-net/sonic-buildimage/issues/16502 and 'dualtor' in topo_name"
+      - "asic_gen in ['th5']"
```
Comment (r12f) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:60]: will be better to move to per topology condition. i will get the topologies in, and let's switch to them.

Comment (StormLiangMS) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:60]: should we file an issue to skip this one based on an open issue? Then when we support it, we can just close the issue.

Comment (ccroy-arista) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:60]: @r12f I could change to standalone topo for now and we could address additional topos as they come up. What do you think?

Comment (ccroy-arista) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml]: Changed from th5 to standalone topos.

**Review by StormLiangMS — APPROVED** [review]
LGTM

**Pull request merged**